src/meta/src/stream/stream_manager.rs

```diff
@@ -64,6 +64,8 @@ pub struct CreateMaterializedViewContext {
     pub table_id_offset: u32,
     /// Internal TableID to Table mapping
     pub internal_table_id_map: HashMap<u32, Option<Table>>,
+    /// Chain fragment set, used to determine when to schedule a chain fragment.
```

Comment:
```suggestion
    /// The set of fragments which need to be colocated with their upstream tables. 
    /// Specifically, they are fragments containing chain nodes.
    ///
    /// They are scheduled in `resolve_chain_node`.
```

Reply:
BTW, what about using `HashMap<FragmentId->UpstreamTableId>`, since this information is needed later? But I'm not sure whether it can save any work. 🥸